src/Microsoft.DncEng.SecretManager/StorageUtils.cs

```diff
@@ -49,6 +49,45 @@ public static async Task<string> RotateStorageAccountKey(string subscriptionId,
             return key.Value;
         }
 
+        public static SharedAccessAccountPermissions AccountPermissionsFromString(string input)
+        {
+            var accessAccountPermissions = SharedAccessAccountPermissions.None;
+            foreach (char ch in input)
+            {
+                accessAccountPermissions |= ch switch
+                {
+                    'a' => SharedAccessAccountPermissions.Add,
```

Comment:
Hrm... this is probably easier to read if we just spend them out, isntead of `rup`, `read | update | processMessages` is a bit clearer in the yaml file.  Since it won't ever change I'd prefer to be a bit more verbose.

Reply:
The azure sdk already has a PermissionsToString method for this type, that is this exact format. This is just the mirror that they didn't write for whatever reason.